frontend/app/routes/protected/multi-channel/sin-confirmation.tsx

```diff
@@ -0,0 +1,221 @@
+import type { ReactNode } from 'react';
+import { useId, useRef } from 'react';
+
+import type { RouteHandle } from 'react-router';
+import { useFetcher } from 'react-router';
+
+import type { ResourceKey } from 'i18next';
+import { useTranslation } from 'react-i18next';
+
+import type { Info, Route } from './+types/sin-confirmation';
+
+import { requireAuth } from '~/.server/utils/auth-utils';
+import { i18nRedirect } from '~/.server/utils/route-utils';
+import { Button } from '~/components/button';
+import { PageTitle } from '~/components/page-title';
+import { AppError } from '~/errors/app-error';
+import { ErrorCodes } from '~/errors/error-codes';
+import { getTranslation } from '~/i18n-config.server';
+import { handle as parentHandle } from '~/routes/protected/layout';
+import { dateToLocalizedText, getLocalTimeZone } from '~/utils/date-utils';
+
+export const handle = {
+  i18nNamespace: [...parentHandle.i18nNamespace, 'protected'],
+} as const satisfies RouteHandle;
+
+export async function loader({ context, request }: Route.LoaderArgs) {
+  requireAuth(context.session, new URL(request.url), ['user']);
+  const { t } = await getTranslation(request, handle.i18nNamespace);
+
+  //TODO: replace with record data (names, city, etc)
+  return {
+    documentTitle: t('protected:sin-confirmation.page-title'),
+    recordDetails: {
+      sinNumber: '123456789',
+      firstName: 'Johnathan',
+      middleNames: ['Joe', 'James'],
+      familyNames: ['Doe', 'Smith'],
+      address: '123 Main St. Suite 4B',
+      postalCode: 'A1A 1A1',
+      city: 'City',
+      province: 'Province',
+    },
+  };
+}
+
+export function meta({ data }: Route.MetaArgs) {
+  return [{ title: data.documentTitle }];
+}
+
+export async function action({ context, request }: Route.ActionArgs) {
+  requireAuth(context.session, new URL(request.url), ['user']);
+
+  const formData = await request.formData();
+  const action = formData.get('action');
+
+  switch (action) {
+    case 'finish': {
+      throw i18nRedirect('routes/protected/request.tsx', request); //TODO: update redirect to proper page
+    }
+    default: {
+      throw new AppError(`Unrecognized action: ${action}`, ErrorCodes.UNRECOGNIZED_ACTION);
+    }
+  }
+}
+
+export default function SinConfirmation({ loaderData, actionData, params }: Route.ComponentProps) {
+  const { t } = useTranslation(handle.i18nNamespace);
+  const fetcherKey = useId();
+  const fetcher = useFetcher<Info['actionData']>({ key: fetcherKey });
+  const isSubmitting = fetcher.state !== 'idle';
+  const contentRef = useRef<HTMLDivElement>(null);
+  const recordDetails = loaderData.recordDetails;
+  const { dateEn, dateFr } = dateToLocalizedText(getLocalTimeZone());
```

Comment:
I think you should use `serverEnvironment.BASE_TIMEZONE` here. Using the local timezone could create a discrepancy between the logged/audited date vs what's printed on the confirmation page.

Actually, since this is a client-side component, we will have to move the base timezone to the client environment. If you want, I can submit a PR to do that.